compiler/rustc_codegen_cranelift/src/value_and_place.rs

```diff
@@ -674,6 +674,14 @@ impl<'tcx> CPlace<'tcx> {
         }
     }
 
+    pub(crate) fn place_transmute_type(
+        self,
+        fx: &mut FunctionCx<'_, '_, 'tcx>,
+        ty: Ty<'tcx>,
+    ) -> CPlace<'tcx> {
+        CPlace { inner: self.inner, layout: fx.layout_of(fx.monomorphize(ty)) }
```

Comment:
can you move the `monomorphize` out of `place_transmute_type` and add a comment that the `ty` has to be monomorphized? It's otherwise possible to incorrectly `monomorphize` twice by accident.